org.sil.Bloom.yml

```
@@ -0,0 +1,647 @@
+                # absent.
+                - "/lib/mono/msbuild"
+                - "/lib/mono/xbuild"
+            sources:
+                - type: file
+                  url: http://linux.lsdev.sil.org/ubuntu/pool/main/m/mono5-sil-msbuild/mono5-sil-msbuild_15.8+xamarinxplat.2018.07.31.22.43-0xamarin5+ubuntu1804b1_all.deb
+                  sha256: 758274d8bc438f325fc5fc808c0482ab50eb2bdaf02f61747888e190760753de
+          - name: libgdiplus
+            build-options:
+                prefix: /app
+                env:
+                    MONO_PREFIX: /app
+            config-opts:
+                - --with-pango
+            cleanup:
+                - "/include"
+                - "*.a"
+            sources:
+                - type: git
+                  url: https://github.com/sillsdev/libgdiplus
+                  tag: merge/5.6.0
+                  commit: b9c581ba87da89651d5ec56c5da0960ab739a63a
+            modules:
+                - name: giflib
+                  buildsystem: simple
+                  build-commands:
+                      # Skip building documentation
+                      - perl -pi -e 's#doc/Makefile##' configure.ac
+                      - ./autogen.sh --prefix=/app
+                      - make
+                      - make install
+                  cleanup:
+                      - "/include"
+                      - "*.a"
+                  sources:
+                      - type: git
+                        url: https://github.com/mirrorer/giflib.git
+                        commit: fa37672085ce4b3d62c51627ab3c8cf2dda8009a
+                        # Note that libgdiplus needs giflib to have GifQuantizeBuffer.
+                        # Some giflibs need adjustment to have that built in.
+          - name: xchm
+            # Help file viewer
+            cleanup:
+                - "/include"
+                - "*.a"
+                - "/share/icons"
+                - "/share/applications"
+            sources:
+                - type: git
+                  url: https://github.com/rzvncj/xCHM
+                  # Version 1.31 is used in Ubuntu 20.04.
+                  tag: "1.31"
+                  commit: e983d11fd546c961501f736b2c6f93178c92fa97
+                - type: patch
+                  # xchm would otherwise need to write to files ~/.xchmXXXXXX and
+                  # ~/.xchm. This won't be needed after upgrading to a newer xchm
+                  # version and wxWidgets v3.1.1+.
+                  path: patches/xchm/no-config.patch
+            modules:
+                - name: chmlib
+                  cleanup:
+                      - "/include"
+                      - "*.a"
+                  sources:
+                      - type: git
+                        url: https://github.com/jedwing/CHMLib
+                        # Latest commit, from 2009-07-03.
+                        commit: 2bef8d063ec7d88a8de6fd9f0513ea42ac0fa21f
+                      - type: script
```

I think you meant 
```suggestion
                      - type: shell
```

otherwise you are missing the dest-filename.

---

changed to shell